ha_lineairdb.cc

```diff
@@ -814,19 +814,33 @@ std::string ha_lineairdb::get_current_key() {
   return std::string(current_key_.ptr(), current_key_.length());
 }
 
+std::array<char, BYTE_BIT_NUMBER> ha_lineairdb::interpret_buf_flag(uchar flag) {
+  std::array<char, BYTE_BIT_NUMBER> byte_nullflag = {};
+  for(int i = 0; i < 8; i++) {
+    byte_nullflag[i] = (flag >> i) & 1;
+  }
+  return byte_nullflag;
```

Comment:
Does this function take a 1-byte bitmap and return an 8-byte array? 
And is the input equivalent to the output, so is it just a helper function?
If so, I think it is inefficient for memory. 
I think that we can use inline functions as follows to do what we want to do.

```c++
inline bool is_a_bit_is_up(uchar& flag, size_t idx){
  return (flag >> idx) & 1;
}
```